templates/.github/workflows/go-check.yml

```diff
@@ -13,7 +13,7 @@ jobs:
           submodules: recursive
       - uses: actions/setup-go@v2
         with:
-          go-version: "1.17.x"
+          go-version: "${{ go.versions[-1] }}"
```

Comment:
Does this work? Won't it panic? ;)

Reply:
I forgot to stage my changes 🤦  Added them in a new commit now so that the comment history makes sense: https://github.com/protocol/.github/pull/280/commits/9bd61de7c19ceb2be6e0f13ea180c408330c280a

Now it reads:
```
go-version: ${{{ config.go.versions[-1] }}}
```

`[-1]` is a valid reference to the last element in `jq` which is what the templating engine I proposed uses. Sadly, it is not in GitHub Actions Expressions.